others/sub_array_with_given_sum.cpp

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2020 @author omkarlanghe
+ *
+ * @file
+ * Given an unsorted array A of size N of non-negative integers,
+ * find a continuous sub-array which adds to a given number S, If not found
+print -1.
+ *
+ * Input:
```

Comment:
Please elaborate on the input. What is the first line with only one value? What is the second line with two values?

Reply:
Change done as per the above review comment.